api/sessions_api.go

```diff
@@ -157,8 +157,6 @@ sessionLoop:
 				}
 			}
 
-			// attacker turn is set to false
-			resp.Payload.IsTurn = false
```

Comment:
is this intentional?

Reply:
So bizarre that i don't see it on my end!